Cargo.toml

```diff
@@ -18,7 +19,7 @@ members = [
   "risc0/zkvm/receipts",
   "xtask",
 ]
-exclude = ["tools/crates-validator"]
+exclude = ["external/substrate", "tools/crates-validator"]
```

Comment:
I think we should drop this, I'd like for a normal `cargo test` to include this test.